piecash/core/session.py

```diff
@@ -334,6 +334,11 @@ def create_book(
 
     # create commodities and initial accounts
     from .account import Account
+    import locale
```

Comment:
the "import locale" can be put at the top of the file (not in the function).
the import just above in line 336 is in the function to avoid circular imports

Reply:
To avoid circular imports issues